runtime/src/lib.rs

```diff
@@ -721,12 +695,10 @@ impl membership_supernodes::Config for Runtime {
 }
 
 parameter_types! {
```

Comment:
why `X1(Parent)` and not `MultiLocation::parent()` for `RelayLocation` https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/runtime/src/lib.rs#L303

Reply:
@ltfschoen You are pointing to latest master branch. I followed this https://github.com/substrate-developer-hub/substrate-parachain-template/blob/polkadot-v0.9.9/runtime/src/lib.rs#L308